coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,19 @@
-class InvalidFilterException(Exception):
-    pass
+class InvalidFilterException(BaseException):
```

Comment:
No please keep it as an `Exception`, `BaseException` is used for "special" exceptions like `KeyboardInterrupt`.